src/App.js

```diff
@@ -1,16 +1,17 @@
 import { Fragment } from 'react';
+import './index.css';
 
 import Header from './components/header/Header';
 import BookList from './components/books/BookList';
 import Form from './components/form/Form';
 
 function App() {
   return (
-    <Fragment>
+    <>
       <Header />
       <BookList />
       <Form />
-    </Fragment>
```

Comment:
- [ ] Also, you should use `Route` and `NavLink` to navigate through your Books and Categories.

Reply:
This change has been done well.